### DISCUSSION TITLE
Add CodeCov coverage badge

### DISCUSSION THREAD

**Discussion by Trusted97** [phasync/phasync#32] [Ideas]
Hi @frodeborli 
In the latest MR we've add CI and a job that upload codecoverage on [CodeCov](https://about.codecov.io/tool/github-actions/), you should be able to retrieve a token and also a badge that could we add for improve readibility on README and tell to the user the actual code coverage :)

[Here](https://github.com/phasync/phasync/actions/runs/9498132411/job/26176189408?pr=31#step:10:59) you could find the error due to the assence of CODECOV_TOKEN

**Comment by frodeborli** [comment]
I have added the badge. Not sure where to put the CODECOV_TOKEN or how to send it to you, but it seems to be correct. It shows I have a job to do on the testing side. Things I never felt was very important to test. :)

**Comment by frodeborli** [comment]
@Trusted97 I got codecov working with the secret and I did some work to increase the coverage.

**Reply by Trusted97** [reply]
Great! I'll start working to increase code coverage too, after that i think we should begin implement PHPStan for static code analysis WDYT?

**Comment by frodeborli** [comment]
I think phpstan might be positive for others, but I already use a language server and don't expect it to make a difference. Still, I see the value and agree.

Sendt fra Outlook for iOS<https://aka.ms/o0ukef>
________________________________
Fra: Gianluca ***@***.***>
Sendt: Monday, June 17, 2024 10:17:19 AM
Til: phasync/phasync ***@***.***>
Kopi: Frode Børli ***@***.***>; Mention ***@***.***>
Emne: Re: [phasync/phasync] Add CodeCov coverage badge (Discussion #32)


Great! I'll start working to increase code coverage too, after that i think we should begin implement PHPStan for static code analysis WDYT?
